docs/sources/operations/storage/tsdb.md

```diff
@@ -0,0 +1,77 @@
+---
+title: TSDB
+---
+# TSDB Index Store
```

Comment:
Same comment as above, spell out TSDB (Note that https://www.acronymfinder.com/ does not yet list Time Series Database for TSDB, so users might struggle with this one if we don't define it for them).

Reply:
Suggested revision - Time series database index store.